newsroom/press-resources/index.html

```diff
@@ -24,9 +24,13 @@ <h1 class="press-section_title press-contacts_title">Press contacts</h1>
 
         <p class="press-section_intro press-contacts_intro">
             We are committed to engaging with the public and members of the media regularly
-            with accurate and up-to-date information on our work. News media should use
-            this information to reach our Office of Communications. For other questions,
-            see <a href="/contact-us/">Contact Us</a>.
+            with accurate and up-to-date information on our work.
+        </p>
+        
+        <p class="press-section_intro press-contacts_intro">
```

Comment:
Not really something I can speak to; I just copied the one that was already there. Want me to go ahead and get rid of them?